L1Trigger/Phase2DAQProducer/BuildFile.xml

```diff
@@ -0,0 +1,8 @@
+<flags CXXFLAGS="-Wno-unused-but-set-variable"/>
```

Comment:
Why do we need the no-unusd-but-set option?

Reply:
we don't. Probably got a compilation error at some point and included that. Will delete.

Reply:
done.